tests/resources/pypi_deps/ubuntu20_runtime_2021.3.json

```diff
@@ -23,7 +19,6 @@
         "numpy >= 1.11.3; python_version>=\"3.6\"\n"
     ],
     "requirements": [
-        "/opt/intel/openvino/data_processing/dl_streamer/requirements.txt",
```

Comment:
can we add exception for opencv, without removing requirements.txt? Because in someday, they can add some other pypi pkg.